sql/src/test/java/io/crate/integrationtests/SQLTransportIntegrationTest.java

```diff
@@ -134,6 +135,16 @@ public void run() {
         }
     }
 
+
+    public void runJobContextReapers() throws Exception{
+        Class<?> innerClass = Class.forName("io.crate.jobs.JobContextService$Reaper");
```

Comment:
maybe instead of using the reaper you could call doStop/doStart on the JobContextService?